browser/services/data.js

```diff
@@ -53,6 +53,27 @@ class InstallerDataService {
     }
   }
 
+  transformation(requirements) {
+    var obj = {};
+    for(var object in requirements) {
+      obj[object] = {};
+      for(var platform in requirements[object]) {
+        if(platform === 'platform') {
+          for(var operatingsystem in requirements[object][platform]) {
+            if(process.platform){
```

Comment:
Does this need to be a conditional? There doesn't seem to be an alternative to this (and if process.platform is not defined, than something else will fail anyway).